mro/__init__.py

```diff
@@ -1,23 +1,31 @@
-﻿
-import mro.connection
+﻿import mro.connection
 import mro.data_types
 import mro.table
 import mro.sqlite
+import mro.custom_types
 
 
 def disconnect():
     mro.connection.disconnect()
 
 
-def load_database(connection_function, exclude_tables=[]):
-
+def load_database(connection_function, hooks=None):
+    print("***********INITIALISING DATABASE************")
     mro.connection.set_connection_function(connection_function)
+    mro.connection.set_on_reconnect(init_db)
+    mro.connection.set_hooks(hooks)
     connection = mro.connection.connection
+    init_db(connection)
+    if hooks is not None:
+        for hook in hooks:
+            hook()
+
 
+def init_db(connection):
     if connection.__class__.__module__ == 'sqlite3':
         tables = sqlite._load_sqllite_db(connection)
     else:
-        tables = _load_standard_db(connection, exclude_tables)
```

Comment:
exclude tables was a hack that was put in as custom types didnt work, removed this as part of this work.